packages/stats/stats.ts

```diff
@@ -33,10 +33,5 @@ export function timeAsync<T>(
    statName: string,
    promiseFunc: () => Promise<T>
 ): Promise<T> {
-   const t = Date.now();
-   const result = logAsync(statName, promiseFunc);
```

Comment:
Did you intentionally drop this `logAsync` call? I thought you wanted to log it and send stats.

If we don't want to log, we could just `export stats.asyncTimer` from this module, since the wrapper doesn't really add anything. But not worth blocking on if we're not making any other changes.

Reply:
Oh snap, you're right. I did drop that on accident. I noticed that the `asyncTimer()` removed the need for much of this code, but I went too far.  I'll make a follow up.